tools/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+PROJECT(vrp_drawer)
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_EXTENSIONS OFF)
+set(SOURCE solution_drawer.cpp)
+set(CMAKE_CXX_FLAGS "-O2 -march=native -std=c++14")
```

Comment:
oh, those are not needed here. they must be derived from top-level CMake that has those already:
https://github.com/hardnp/vehicle_routing_problem/blob/master/CMakeLists.txt

Reply:
same for `CMAKE_CXX_STANDARD`, etc.